.travis.yml

```diff
@@ -21,15 +21,26 @@ matrix:
   include:
     - compiler: gcc
       env: MODE=build-gcc
+      arch: amd64
+    - compiler: gcc
+      env: MODE=build-gcc
+      arch: ppc64le 
     - compiler: gcc
       env: MODE=build-gcc ENABLE_UCS4=--enable-ucs4
+      arch: amd64
+    - compiler: gcc
+      env: MODE=build-gcc ENABLE_UCS4=--enable-ucs4
+      arch: ppc64le
     - compiler: i686-w64-mingw32-gcc
       env: MODE=build-mingw ENABLE_UCS4=--enable-ucs4 ARCH=32
     - compiler: x86_64-w64-mingw32-gcc
       env: MODE=build-mingw ENABLE_UCS4=--enable-ucs4 ARCH=64
     - compiler: gcc
       env: MODE=build-js
     - env: MODE=check-format
+      arch: amd64
```

Comment:
I don't think we need to run `check-format` on more than one architecture

Reply:
Ok , will do that too. :)
Thanks !!